examples/internal/helloworld/helloworld.swagger.json

```diff
@@ -15,918 +15,948 @@
   "produces": [
     "application/json"
   ],
-  "paths": {
-    "/say/boolval/{boolVal}": {
-      "get": {
-        "operationId": "Greeter_SayHello5",
-        "responses": {
-          "200": {
-            "description": "A successful response.",
-            "schema": {
-              "$ref": "#/definitions/helloworldHelloReply"
+  "paths": [
+    {
+      "Path": "/say/boolval/{boolVal}",
```

Comment:
This doesn't look quite right, I think we want the structure to remain the same, right? We still want this to be a key, not a value.

Reply:
I'm so sorry about missing this, this was a silly mistake on my part. I've pushed a new commit to try and address this problem. I've implemented some custom marshallers for the `openapiPathsObject` for both yaml and json to keep using the ordered data structure, but also preserve the desired swagger file outputs.